backend/curriculum_tracking/management/commands/export_project_urls.py

```diff
@@ -0,0 +1,29 @@
+from django.core.management.base import BaseCommand
+from core.models import Team
+from curriculum_tracking.models import RecruitProject
+import csv
+
+
+def get_email_and_url(team):
+    results = []
+    for user in team.active_users:
+        project = RecruitProject.objects.filter(recruit_users__in=[user]).first()
+        results.append([user.email, project and (project.link_submission or project.git_url)])
+    return results[0]
```

Comment:
why are you only returning the first one?

Reply:
The returned object is a list of lists, where each inner list is a row that we'll write to the csv. So I'm getting just the actual lists, rather than the whole thing.